workspace/pipeline/pln_document_metadata_main.json

```diff
@@ -18,6 +18,12 @@
 						"dependencyConditions": [
 							"Completed"
 						]
+					},
+					{
+						"activity": "pln_service_bus_nsip_document",
+						"dependencyConditions": [
+							"Succeeded"
```

Comment:
Do we think succeeded is correct here? The other 2 pipelines have "completed" as the condition. Should all 3 be the same?

Reply:
Updated so all 3 are on succeeded. On completion is causing failures to not be raised to parent pipelines